packages/kujira-std/src/ica.rs

```diff
@@ -0,0 +1,159 @@
+use cosmwasm_schema::{cw_serde, serde::Serializer};
+use cosmwasm_std::Binary;
+
+#[cw_serde]
+pub enum IcaSudoMsg {
+    IcaRegisterCallback(IcaRegisterCallbackData),
+    IcaTxCallback(IcaTxCallbackData),
+}
+
+#[cw_serde]
+pub struct IcaRegisterCallbackData {
+    pub connection_id: String,
+    pub account_id: String,
+    pub callback: Option<Binary>,
+    pub result: IcaResult,
+}
+
+#[cw_serde]
+pub struct IcaTxCallbackData {
+    pub connection_id: String,
+    pub account_id: String,
+    pub sequence: u64,
+    pub callback: Option<Binary>,
+    pub result: IcaResult,
```

Comment:
I'm not sure the same `IcaResult` can be applied here? Does the `IcaTxCallbackData` still use `IcaOpenVersion`?

Reply:
reverted to Binary for this one, but we might want to get clever and have the contract define the type so that it deserialises automatically